samples/bluetooth/public_broadcast_sink/src/main.c

```diff
@@ -0,0 +1,444 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <errno.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/bluetooth/audio/pacs.h>
+#include <zephyr/bluetooth/audio/bap_lc3_preset.h>
+#include <zephyr/bluetooth/audio/pbp.h>
+
+#define AVAILABLE_SINK_CONTEXT  (BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | \
+				 BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | \
+				 BT_AUDIO_CONTEXT_TYPE_MEDIA | \
+				 BT_AUDIO_CONTEXT_TYPE_GAME | \
+				 BT_AUDIO_CONTEXT_TYPE_INSTRUCTIONAL)
+
+#define SEM_TIMEOUT K_SECONDS(10)
+#define PA_SYNC_SKIP         5
+#define SYNC_RETRY_COUNT     6 /* similar to retries for connections */
+#define INVALID_BROADCAST_ID 0xFFFFFFFF
+
+static bool pbs_found;
+static uint8_t meta[50];
```

Comment:
Would it make sense to use `BT_AUDIO_CODEC_CAP_MAX_METADATA_SIZE` for the metadata size?